docs/dynamic-plugins/installing-plugins.md

```diff
@@ -36,6 +36,61 @@ While the plugin's default configuration comes from the `dynamic-plugins.default
 
 Note: The plugin's default configuration typically references environment variables, and it is essential to ensure that these variables are set in the Helm chart values or the Operator configuration.
 
+## Using a Catalog Index Image for Default Plugin Configurations
+
+RHDH supports loading default plugin configurations from an OCI container image. This feature allows you to maintain centralized plugin configurations that can be updated independently of the RHDH container image.
+
+When the `CATALOG_INDEX_IMAGE` environment variable is set, the `install-dynamic-plugins` init container will:
+
+1. Download and extract the specified OCI image
+2. Look for a `dynamic-plugins.default.yaml` file within the image
+3. Use this file as the primary source for default plugin configurations
+4. Automatically override the embedded `dynamic-plugins.default.yaml` file if present
+
+### Configuring the Catalog Index Image
+
+Set the `CATALOG_INDEX_IMAGE` environment variable in the `install-dynamic-plugins` init container to specify the OCI image containing your plugin catalog:
+
+```yaml
+# Example using RHDH Operator (Kubernetes/OpenShift)
+apiVersion: rhdh.redhat.com/v1alpha4
+kind: Backstage
+metadata:
+  name: my-backstage
+spec:
+  application:
+    extraEnvs:
+      envs:
+        - name: CATALOG_INDEX_IMAGE
+          value: "quay.io/rhdh/plugin-catalog-index:1.9"
+          containers: ["install-dynamic-plugins"]
+```
+
+```yaml
+# Example using Helm chart values
+global:
+  dynamic:
+    catalogIndex:
+      image: "quay.io/rhdh/plugin-catalog-index:1.9"
+```
+
+The Helm chart automatically templates this value into the `CATALOG_INDEX_IMAGE` environment variable. To update the catalog index, modify this value and run `helm upgrade`.
```

Comment:
This mentions a behavior not implemented yet in the Helm Chart. The documentation here should reflect the current behavior.
So for now (until this is implemented), users would need to copy and update the `install-dynamic-plugins` init container definition (https://github.com/redhat-developer/rhdh-chart/blob/main/charts/backstage/values.yaml#L186-L235) in their custom values file and insert the `CATALOG_INDEX_IMAGE` env var.
Could you reflect this here?
Once the `global.dynamic-catalogIndex` field is handled in the Helm Chart, we would consider updating this doc.